wtxmgr/db.go

```diff
@@ -241,35 +257,63 @@ func keyBlockRecord(height int32) []byte {
 }
 
 func valueBlockRecordEmpty(block *BlockMeta) []byte {
-	v := make([]byte, 46)
+	v := make([]byte, 47)
 	copy(v, block.Hash[:])
 	byteOrder.PutUint64(v[32:40], uint64(block.Time.Unix()))
 	byteOrder.PutUint16(v[40:42], block.VoteBits)
-	byteOrder.PutUint32(v[42:46], 0)
+	byteOrder.PutUint32(v[43:47], 0)
```

Comment:
Fair enough about it being out of scope for this PR.  I did see the numbers in the old portion of the code too so I wasn't sure about complaining about it relative to this PR.  But I would love constants for it in the long run.